README.md

```diff
@@ -115,3 +115,22 @@ Since the library is published in the GitHub Packages repository, you need to ad
   <url>https://maven.pkg.github.com/rcardin/kafkaesque</url>
 </repository>
 ```
+
+#### Configuration
+
+_Kafkaesque_ also supports internal producer and consumer configuration via an `application.properties`, `application.conf`  or `application.json` configuration file.
```

Comment:
What about YAML files? Where should the file be placed?

Reply:
Yes @rcardin, as you mentioned, YAML is not supported yet by the library. I have been playing around this lib as well as [Apache Commons Configuration](https://commons.apache.org/proper/commons-configuration/index.html), the latter supports YAML but I had some issues, I will revisit it in the next iteration.